dev/run_dev.R

```diff
@@ -12,7 +12,7 @@ golem::detach_all_attached()
 golem::document_and_reload()
 
 # Run the application 
-run_app(pre_auth_user = TRUE)
+run_app()
```

Comment:
Why did you remove the `pre_auth_user = TRUE` option?

Reply:
@Jeff-Thompson12 I'm actually not sure. I might have been testing something? I've returned in a recent commit.